### ISSUE TITLE
[MOBILE] 

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#7839]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

On After a Board Appeal — [https://preview-prod.vfs.va.gov/preview?nodeId=3013](https://preview-prod.vfs.va.gov/preview?nodeId=3013), under the heading "Add new and relevant evidence" there are two links on individual lines, but they are very close together. **Consider** what options are available to have 1.5rem / 24px space between interactive elements.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a touch interface user, I want to select the link I want without accidentally hitting another too close link.

## Environment

* Operating System: all
* Browser: all
* Screenreading device: all
* Server destination: Devshop?/Prod?

## Steps to Recreate

1. Enter https://preview-prod.vfs.va.gov/preview?nodeId=3013 in browser, preferably on a mobile or touch device
2. Touch either of the two links referenced in the description
3. Now, consider someone with the [fat finger issue](https://www.dictionary.com/browse/fat-finger), hand weakness or tremors

## Possible Fixes (optional)

In other instances on the site, we've adjusted copy a bit to ensure sufficient space. In this instance, it may be as simple as placing each link in a bullet, if that puts sufficient space. 

## WCAG or Vendor Guidance (optional)

* [Carnegie Museums - Links, B](http://web-accessibility.carnegiemuseums.org/content/buttons/) - "Make clickable elements large, and put space between them."
* [Sitepoint - making accessible links](https://www.sitepoint.com/15-rules-making-accessible-links/) - "It is still important that there is space between adjacent links."

## Screenshots or Trace Logs

![image](https://user-images.githubusercontent.com/57469/78941883-868fd180-7a86-11ea-8a8a-74997bf2a864.png)

**Comment by kelsonic** [comment]
This is in a `paragraphs/wysiwyg` (entityID is 8811) so will need to be modified in Drupal I believe:

![image](https://user-images.githubusercontent.com/12773166/82693132-2f9d2080-9c1e-11ea-88b4-fa495c20c846.png)

**Comment by jenstrickland** [comment]
@lwalsh617 If you'd like to hop on a call with me with the editor open we can pair-up to address it. Other pages  put each link in a separate p tag, or use a list.

**Comment by lwalsh617** [comment]
@jenniferlee-dsva @RLHecht 
Lengthy Slack discussion on this. Two possible solves. 
**Option 1 is adding spacing**, like in this example:
![Screenshot 1_wider spacing](https://user-images.githubusercontent.com/53535663/84817622-44818f80-afe3-11ea-941a-e984cb0267e4.png)
**Option 2 is adding copy:** 
![Screen Shot 2_Copy added](https://user-images.githubusercontent.com/53535663/84817596-3a5f9100-afe3-11ea-8da7-9ebe98da158c.png)
Please let me know which way to go. Thanks!

**Comment by lwalsh617** [comment]
Per @bethpottsVADEPO we are not going to make this change now, but as part of a site-wide effort.

**Comment by laflannery** [comment]
@coforma-terry I was going through backlog items again and have confirmed this is content for the CAIA team to update, not code for PW. Would you all like me to file a new intake ticket for your team or are you able to add this ticket to your backlog/pipelines?

**Comment by megzehn** [comment]
Thanks @RLHecht and @laflannery. I added a line between the 2 links and published the changes. 

I'm guessing we have this issue on a few other pages. Do either of you know if there's an easy way to scan for this issue on other pages? Or if we're tracking these issues anywhere else? Thanks so much!

**Comment by laflannery** [comment]
Thanks @megzehn!

Unfortunately we don't have a way to find all of these instances. I think we will just have to be aware of this and note them as we come across them. I do know that I don't have any others in my own backlog but I will keep an eye out in my day to day and make sure I add intake tickets for CAIA if I come across others. Does that work?
